modules/dnn/src/ie_ngraph.cpp

```diff
@@ -48,6 +48,43 @@ ngraphWrappers(const std::vector<Ptr<BackendWrapper> >& ptrs)
     return wrappers;
 }
 
+ov::element::Type cvTypeToOvType(MatType cvType)
```

Comment:
Method is called differently for Mat using `.type()` or `.depth()`. This is not critical for dnn because everything is 1-channel but I suggest to add an overload
```cpp
ov::element::Type cvTypeToOvType(const cv::Mat& m) {
    return cvTypeToOvType(m.depth());
}
```

Reply:
What the point to add a second cvTypeToOvType function and don't use it?

Reply:
Why not use? I mean add and replace every invocation on Mat type:
```cpp
cvTypeToOvType(inputs[i].depth()) -> cvTypeToOvType(inputs[i])
cvTypeToOvType(m.type()) -> cvTypeToOvType(m)
cvTypeToOvType(blobs[0].type()) -> cvTypeToOvType(blobs[0])
```